lesson_12/structs_java/structs_app/src/main/java/com/codedifferently/lesson12/Lesson12.java

```diff
@@ -7,6 +7,35 @@ public class Lesson12 {
    * https://github.com/yang-su2000/Leetcode-algorithm-practice/tree/master/3062-winner-of-the-linked-list-game
    */
   public String gameResult(ListNode head) {
-    return null;
+    int evenCounter = 0; // Keeps track of even score
+    int oddCounter = 0; // Keeps track of odd score
+
+    while (head != null
+        && head.next != null) { // Goes through the Linked List until theres no more pairs
+      int evenValue =
```

Comment:
I'm surprised the linter didn't complain. The variable assignment should be on a single line.